src/js/lib/monster.util.js

```diff
@@ -1245,6 +1234,48 @@ define(function(require) {
 			);
 	}
 
+	/**
+	 * Generates a string of `length` random characters chosen from either a
+	 * preset or a custom string of characters.
+	 * @param  {Number} length  Number of characters to include.
+	 * @param  {String} pPreset Characters to choose from.
```

Comment:
Javascript primitive types start with lowercase: https://github.com/airbnb/javascript#types
Also, is there no problem if this parameter name differs from the one that is specified in the MD document?

Reply:
The section you are referring to details how primitive values are stored into variables (deep instead of shallow copy), I don't see a reference to lower casing.

Arguments that start with a lower case `p` usually mean they are optional, useful for type checking and allow us to declare a variable with the actual name:
```javascript
function getLength(array, pMultiplier) {
    // checks on mandatory `array` argument
    // ...

    // check on optional argument
    var multiplier = _.isUndefined(pMultiplier)
        ? 1
        : pMultiplier;

    // now you can use `multiplier` in any cases
}
```
We usually use this technique to assign default values to optional arguments.

Since it does not really matter for developers to know that's how optional arguments are handled internally,  in the doc we remove the leading `p` from optional arguments and simply display the name.

There is other ways to specifically declare optional arguments in the doc:

- in the function syntax section: anything between `[]` is optional
- in the parameters section: the `Required` column

P.S.: Please try to refer to [our style guide](https://docs.2600hz.com/ui/docs/javascriptStyleGuide/) instead of Airbnb's as it contains modifications to match Monster UI's codebase.